<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Imager</class>
 <widget class="QMainWindow" name="Imager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1711</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ClickableLabel" name="bigLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel1">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>30</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel2">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>30</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel3">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>30</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel5">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>165</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel6">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>165</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>165</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel8">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>300</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel9">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>300</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resLabel7">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>300</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1711</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelect_Folder"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_Folder">
   <property name="text">
    <string>Select Folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
